examples/tv-app/android/java/AppImpl.cpp

```diff
@@ -403,6 +403,30 @@ Access::Privilege ContentAppFactoryImpl::GetVendorPrivilege(uint16_t vendorId)
     return Access::Privilege::kOperate;
 }
 
+constexpr ClusterId kClusterIdDescriptor = 0x001d;
+constexpr ClusterId kClusterIdOnOff      = 0x0006;
+constexpr ClusterId kClusterIdWakeOnLAN  = 0x0503;
+// constexpr ClusterId kClusterIdChannel             = 0x0504;
+// constexpr ClusterId kClusterIdTargetNavigator     = 0x0505;
+constexpr ClusterId kClusterIdMediaPlayback = 0x0506;
+// constexpr ClusterId kClusterIdMediaInput          = 0x0507;
+constexpr ClusterId kClusterIdLowPower        = 0x0508;
+constexpr ClusterId kClusterIdKeypadInput     = 0x0509;
+constexpr ClusterId kClusterIdContentLauncher = 0x050a;
+constexpr ClusterId kClusterIdAudioOutput     = 0x050b;
+// constexpr ClusterId kClusterIdApplicationLauncher = 0x050c;
+// constexpr ClusterId kClusterIdAccountLogin        = 0x050e;
+
+std::list<ClusterId> ContentAppFactoryImpl::GetAllowedClusterListForStaticEndpoint(EndpointId endpointId, uint16_t vendorId)
```

Comment:
Any reason not to pass productId as well.

Reply:
added productId